griptape/loaders/base_text_loader.py

```diff
@@ -33,19 +33,16 @@ class BaseTextLoader(BaseLoader, ABC):
     embedding_driver: Optional[BaseEmbeddingDriver] = field(default=None, kw_only=True)
     encoding: str = field(default="utf-8", kw_only=True)
 
-    def _text_to_artifacts(self, text: str | Path) -> list[TextArtifact]:
-        artifacts = []
+    def load_collection(self, sources: Sequence[Any], *args, **kwargs) -> dict[str, list[TextArtifact]]:
+        return cast(Any, super().load_collection(sources, *args, **kwargs))
```

Comment:
Can we refine the cast type from `Any` to `dict[str, list[TextArtifact]]`?

Reply:
Sure! (I had chosen `Any` as it is shorter without any functional difference, but I'm fine either way)